clients/js/src/cmds/info/wrapped.ts

```diff
@@ -11,7 +11,7 @@ export const builder = (y: typeof yargs) =>
   y
     .positional("origin-chain", {
       describe: "Chain that wrapped asset came from",
-      choices: CHAIN_ID_OR_NAME_CHOICES,
+      type: "string",
```

Comment:
I liked the chain names listed out in the suggestions, can you produce the same thing as was set previously with just the full list of SDKv2 chain names?

Reply:
If I do that, then the chain names must be spelled correctly with case sensitivity (ie. Solana, Algorand, etc.). The previous version of worm took chains as all lower case only.  This allows users to continue passing in all lower case chain names without having to capitalize the first letter.  But this should really be user driven.  I don't know if the users have a preference?

Reply:
Ah I see what you mean, I agree its important that existing cli invocations keep working.

Could we still provide the chain names as suggestions then coerce them to the new chain name on parse?

there is a coerce option parameter here: https://yargs.js.org/docs/#api-reference-optionkey-opt

Reply:
The choices option guarantees that the input provided must be one of the supplied choices.  If you want the chains listed as lower case then that would enforce that users must enter lower case chain names and couldn't capitalize the first letter.  That could be used with the coerce() option to transform the chain into a Chain.  Another option is that choices lists all the chains in lower case and Chain format.  But that might produce a very large message on error.

Reply:
Recording for posterity.  The agreement we came to was:

1. we keep it as a string like you have now
2. add a chains command to worm that prints out all the chains (can't remember if we have this already)
3. change the description to this:

```describe: "Chain (Ethereum, Solana, etc.). To see a list of supported chains, run `worm chains`",```

Reply:
Changed as agreed.